ui/src/app/views/app/component.html

```diff
@@ -35,26 +33,32 @@
       </clr-dropdown-menu>
     </clr-dropdown>
   </div>
-  <div class="top-nav btn-primary">
-    <button class="btn tab-link"
-        [class.active]="workspacesActive"
-        routerLink="/">My Workspaces</button>
-    <button class="btn tab-link"
-        [class.active]="reviewActive"
-        *ngIf="hasReviewResearchPurpose"
-        routerLink="/admin/review-workspace">Review Workspaces</button>
-    <button class="btn tab-link"
-        [class.active]="reviewActive"
-        *ngIf="hasReviewIdVerification"
-        routerLink="/admin/review-id-verification">Review ID Verifications</button>
-  </div>
-  <div>
-    <!--
-      Angular's builtin Router module will attach specific views here
-      based on clicking routerLink elements (above) or router.navigate calls.
-    -->
-    <router-outlet></router-outlet>
+
+  <div class="content-container">
+    <nav class="sidenav" [style.min-height.px]="sidebarHeight">
+      <section class="sidenav-content">
+        <button class="btn btn-primary nav-link"
+            [class.active]="workspacesActive"
```

Comment:
Does [`routerLinkActive`](https://angular.io/api/router/RouterLinkActive) work here?

Reply:
We tried that, and it didn't work. I can do another quick attempt, but it seemed not to be working before.

Reply:
Right! Yeah, so this doesn't actually deselect the workspaces page when you go to another page, because technically the root route is always active. We could change our URL structure around it to make it work, theoretically?